cli/tools/coverage/mod.rs

```diff
@@ -24,7 +24,9 @@ use regex::Regex;
 use std::fs;
 use std::fs::File;
 use std::io::BufWriter;
-use std::io::{self, Error, Write};
+use std::io::Error;
+use std::io::Write;
+use std::io::{self};
```

Comment:
```suggestion
use std::io;
```

Maybe this should be reported to rustfmt 👀

Reply:
From https://github.com/rust-lang/rustfmt/issues/4681

> Note that `use crate::lexer;` and `use crate::lexer::{self};` are not actually equivalent: The latter only imports the lexer module, while the former looks in all namespaces, so it can import up to 3 items at once, e.g. a function, a module and a macro of the same name.

Reply:
I think just `use std::io;` works in this case though?

Reply:
I missed that this was auto merging, but we can “fix” these as we see them in the future.

Reply:
👍